src/main/java/com/prgrms/board/error/ValidExceptionHandler.java

```diff
@@ -0,0 +1,22 @@
+package com.prgrms.board.error;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.MethodArgumentNotValidException;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+import org.springframework.web.bind.annotation.RestControllerAdvice;
+
+@RestControllerAdvice
+public class ValidExceptionHandler {
```

Comment:
```suggestion
public class InvalidExceptionHandler {
```

마찬가지로 또는 NotValidExceptionHandler가 어떤가요?

Reply:
이 부분도 InValidExceptionHandler로 적용하였습니다~